README.md

```diff
@@ -12,9 +12,9 @@ shipped with Julia v1.0 and above.
 If you want to develop this package do the following steps:
 - Make a fork and then clone the repo locally on your computer
 - In line 2 of the `Project.toml` file (the line that begins with `uuid = ...`), modify the UUID, e.g. change the `44cf...` to `54cf...`.
-- Change the current directory to the Pkg repo you just cloned and start julia with `julia --project`.
+- Change the current directory to the Pkg repo you just cloned and start a julia session (using the [nightly version](https://julialang.org/downloads/nightlies/)) with `julia --project`.
 - `import Pkg` will now load the files in the cloned repo instead of the Pkg stdlib .
-- To test your changes, simply do `include("test/runtests.jl")`.
+- To test your changes, simply run `Pkg.build()` then `include("test/runtests.jl")`.
```

Comment:
alt: which uses [the command that's run in the CI](https://github.com/JuliaLang/Pkg.jl/blob/ff6d645cd8bd18f4d22f6b5a24ff0234cf193fa5/.github/workflows/test.yml#L61)

```suggestion
- To test your changes switch the current directory to the Pkg repo you just cloned and run `julia --project -e 'import Pkg; Pkg.build(); Pkg.test()'` using the julia [nightly version](https://julialang.org/downloads/nightlies/).
```